### ISSUE TITLE
Request upstream width changes

### ISSUE THREAD

**Issue by ianb** [mozilla/side-view#407]
The WebExperiment increased the sidebar width. We should review if people used it (though the metrics were wonky in that area).

Anyway, we should propose increasing the max-width globally, because it just seems sensible.

**Comment by KopfKrieg** [comment]
Yep, definitely increase the max-width (1/3 of the screen width would be optimal for me). In it's current state side-view is literally unusable for me because the width is just too small.
